api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -358,9 +366,9 @@ public Properties getRuntimeProperties() {
 			tempappdir.mkdir(); // turn it into a directory
```

Comment:
apply the same changes in this file as in the Jupiter one. the only difference is of course that you are using the ClassRule and TemporaryFolder. please also take into account my comments on the Jupiter file. they apply here as well